StructureDefinition/Extension-ODS-OrganisationRelationships.StructureDefinition.xml

```diff
@@ -24,10 +39,12 @@
         </discriminator>
         <rules value="open" />
       </slicing>
+      <min value="0" />
```

Comment:
The min value seems unneeded as this will be 0 by default

Reply:
Good point. Can remain for now but can be discussed in DA to determine if there is/should be a policy for this.